tests/system/python/e2e/test_e2e_kafka.py

```diff
@@ -83,7 +83,7 @@ def _configure_and_start_north_kafka(self, north_branch, fledge_url, host, port,
                 "schedule_type": 3,
                 "schedule_day": 0,
                 "schedule_time": 0,
-                "schedule_repeat": 0,
+                "schedule_repeat": 5,
```

Comment:
This means that task never runs/execute before as its type is INTERVAL?
For nightly system tests have we disabled the skip_verify_north_interface?